pkg/csi/service/fcd/vsphere_helper.go

```diff
@@ -31,8 +33,35 @@ import (
 const (
 	NUM_OF_CONNECTION_ATTEMPTS     int = 3
 	RETRY_ATTEMPT_DELAY_IN_SECONDS int = 1
+
+	MINIMUM_SUPPORTED_VCENTER_MAJOR int = 6
+	MINIMUM_SUPPORTED_VCENTER_MINOR int = 5
 )
 
+func APICheck(version string) error {
```

Comment:
I see this is exported, is it going to be used in other packages?

Reply:
Nope... will rename the function. Was complaining that "API" was in lowercase.

Reply:
maybe `checkAPI`?